manim/manim.cfg

```diff
@@ -0,0 +1,141 @@
+# manim.cfg
```

Comment:
Should this be in `manim/files/`?

Reply:
I'm neutral as to where it goes, though having a whole folder for a single file (for now) seems excessive to me perhaps?

Reply:
I would expect `manim.cfg` and the tex template both to go there, although I'm not sure if we need to keep ctex_template.tex. Whatever you think is best is fine by me.